contract/models/abstract_contract.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2014 Angel Moya <angel.moya@domatix.com>
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016-2018 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, fields
+
+
+class AbstractAccountAnalyticContract(models.AbstractModel):
+    _name = 'account.abstract.analytic.contract'
+    _description = 'Abstract Account Analytic Contract'
```

Comment:
Let's use the descriptions to provide better names (while we keep the old model names for compatibility and ease of migration).

```suggestion
    _description = 'Abstract Recurring Contract'
```